src/http_api/errors.rs

```diff
@@ -0,0 +1,19 @@
+use thiserror::Error;
```

Comment:
Ah, I didn't see the `impl` part. I moved `HandlerError` to to a common module because two other modules need to use it, and for sure we can start re-organizing `http_api` module.